### PR TITLE
native quantiles work

### PR DESCRIPTION
**Pull request by vogievetsky**
The long lost code :-)

### CODE REVIEW

**Comment by c0b** [comment]
so this means it require the input array be already sorted in ascending order (like [d3.quantile documented](https://github.com/d3/d3/wiki/Arrays#d3_quantile)) ? could you document it in `docs/expressions.md` as well?

https://github.com/implydata/plywood/blob/master/docs/expressions.md#aggregate-expressions

File: src/datatypes/dataset.ts [hunk 1]
```diff
@@ -565,7 +565,26 @@ module Plywood {
...
     public quantile(exFn: ComputeFn, quantile: number, context: Datum): number {
-      return quantile; // ToDo fill this in
+      var data = this.data;
+      var vs: number[] = [];
+      for (let datum of data) {
+        var v = exFn(datum, context);
+        if (v != null) vs.push(v);
+      }
+
+      vs.sort((a: number, b: number) => a - b);
```
Comment (c0b) [inline src/datatypes/dataset.ts]: this sort means it does not require

File: test/overall/compute.mocha.js [hunk 2]
```diff
@@ -381,6 +381,56 @@ describe("compute native", () => {
...
+      { vOdd:  6, vEven:  6 },
+      { vOdd:  7, vEven:  7 },
+      { vOdd:  8, vEven:  8 },
+      { vOdd:  8, vEven:  8 },
+      { vOdd:  9, vEven: null },
+      { vOdd: 10, vEven: 10 },
+      { vOdd: 13, vEven: 13 },
+      { vOdd: 15, vEven: 15 },
+      { vOdd: 16, vEven: 16 },
+      { vOdd: 20, vEven: 20 }
```
Comment (c0b) [inline test/overall/compute.mocha.js]: but the test case is already sorted

Comment (vogievetsky) [inline test/overall/compute.mocha.js]: Nope it is not required, the data is treated as a set. Will update the docs and the test case.

**Comment by evasomething** [comment]
💧

**Pull request merged**